eyeball_pp/comparators.py

```diff
@@ -115,6 +144,31 @@ def _execute_comparator(
         return OutputFeedback(FeedbackResult.NEUTRAL, reason)
 
 
+def _execute_llm_operation(
+        objective: str,
+        input_variables: dict[str, str],
+        older_checkpoint_response: str,
+        newer_checkpoint_response: str
+) -> OutputFeedback:
+    token_encoding = tiktoken.get_encoding("cl100k_base")
+    token_count = len(token_encoding.encode(
+        older_checkpoint_response + newer_checkpoint_response))
+    COMPARISON_MAX_TOKEN_COUNT = int(os.environ.get('COMPARISON_MAX_TOKEN_COUNT', 6000))
```

Comment:
This needs improvement because technically we should decide based on the size of the entire prompt including inputs and prompt string. But simplified for now.